### ISSUE TITLE
All file downloads result in 'DownloadView' file - unrecognized format

### ISSUE THREAD

**Issue by beaucon7** [google/grr#557]
Hi,
I've had GRR installed for a few months now without issue however recently when we try to obtain/download files the file downloaded is called 'DownloadView' and it is an unrecognized format by Windows...

I can open the file with a text editor to see some content that appears to be a container... never the less the file I'm trying to download (.csv, .txt, etc) is not received.

I'll click download new version, then after it downloads I see the hash and the 'download' button, along with the various file sizes in KB. But when I click download only a 'DownloadView' file is downloaded...

Any ideas?

**Comment by mbushkov** [comment]
A few questions:
1. Which GRR version do you have installed?
1. Does GRR web UI run as a standalone process or behind a reverse proxy (Apache, Nginx)?
1. Did you update/upgrade GRR before the issue started?
1. Do I understand it correctly that the "Download" feature was working before and then stopped working for some reason?
1. Which browser (platform, version) are you using? What happens if you try another browser?

**Comment by beaucon7** [comment]
Which GRR version do you have installed? **I'm not sure. 3.1.0.2 ?? That's what all my agents are labeled as. How should I be able to tell version? I didn't find it in help, or anywhere in the ui..**

Does GRR web UI run as a standalone process or behind a reverse proxy (Apache, Nginx)?**I'm pretty sure it's a standalone process**

Did you update/upgrade GRR before the issue started?**As far as I'm aware no changes were made... **

Do I understand it correctly that the "Download" feature was working before and then stopped working for some reason?**Yes. We installed it, ran through made sure everything was functioning - then we got put on hold for 2 months or so. Here and now, we're good to go... but it's not working. I thought about a rebuild, but didn't want to if it was an 'easy fix'**

Was using Chrome. I tried your suggestion with IE and it had more than a few errors in the UI - none of the panels on the left even loaded... Thinking out loud, a rebuild is probably best..

**Comment by mbushkov** [comment]
I wonder if it's a change in Chrome behavior that may be causing this issue?

GRR version is not displayed in the UI. How did you install GRR - from a DEB package?

Do you think you can try installing the latest GRR release (3.2.1) and check if the error is still there? http://grr-doc.readthedocs.io/en/v3.2.1/installing-grr-server/from-release-deb.html

You don't necessarily have to upgrade your current deployment to check if you still have an error - a test deployment with 1 client is likely enough (you can run a client and a server on a same machine for testing).

If you want to try the new GRR version on your current install, make sure to keep the /etc/grr/server.local.yaml file - then your server will be able to talk to all your current clients. I.e. you can first upgrade the server without upgrading the clients and then upgrade the clients later.

**Comment by beaucon7** [comment]
Alright, so the old instance is now saying the database image is corrupt - so something happened. I've decided to move on and start a fresh build from scratch... but it seems I can't even do that correctly.

For simplicity I'm trying on a local VM first - Installed Ubuntu 16.4.0.3 and whenever I try to install the deb packaged it tells me grr-server package have unmet dependencies: debhelper but it is not installable, python2.7-dev but it is not installable, rpm but it is not a installable...

I've tried to troubleshooot this issue to no availa. So I then tried to install on Ubuntu 14 and encounter hundreds of 'release * for * was not found'...

Following this guide: http://grr-doc.readthedocs.io/en/v3.2.1/installing-grr-server/from-release-deb.html#

Any advice is greatly appreciate. 
Thanks.
